client/reader/stream/index.jsx

```diff
@@ -483,7 +482,8 @@ export default localize(
 	connect(
 		state => ( {
 			blockedSites: getBlockedSites( state ),
+			totalSubs: getReaderFollows( state ).length,
```

Comment:
nope! itemCount is pretty tricky and we need to redo that reducer in a separate PR.  Its easy enough to get the count from the api but then how do we keep it up to date as the calypso is running and feeds are subbed/unsubbed?